### PR TITLE
Functional test changes for OBM refactoring

### PR DESCRIPTION
**Pull request by dalebremner**
@RackHD/corecommitters @srinia6 @BillyAbildgaard @zyoung51 @jlongever 

Dependent on:
RackHD/on-core#164
RackHD/on-http#312
RackHD/on-tasks#243
RackHD/on-http#348

* Requires PyPI update of on-http client libraries

### CODE REVIEW

**Comment by srinia6** [comment]
+1

File: test/tests/api/v1_1/obm_settings.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def _set_ipmi(self, uid):
             }
             LOG.info('Creating ipmi obm-settings for node {0} \n {1}'.format(uid,setting))
             try:
-                Nodes().nodes_identifier_patch(uid,setting)
```
Comment (brianparry) [inline test/tests/api/v1_1/obm_settings.py]: @dalebremner Shouldn't nodes  patch should still let you change OBM settings?

Comment (dalebremner) [inline test/tests/api/v1_1/obm_settings.py]: I don't think it does... @srinia6 can you verify?

Comment (dalebremner) [inline test/tests/api/v1_1/obm_settings.py]: You are correct @brianparry ... I removed this change...

**Comment by brianparry** [comment]
test this please

**Comment by brianparry** [comment]
test this please

**Comment by jlongever** [comment]
test this please

**Comment by brianparry** [comment]
The OBMs settings change broke Redfish discovery.  I'll merge this change and put up a new PR with the fix.  The functional tests should pass in the new PR.

**Comment by brianparry** [comment]
Failing tests are fixed here: https://github.com/RackHD/on-tasks/pull/263

**Comment by jlongever** [comment]
test this please

**Pull request merged**